lambda/tile_ingest_lambda.py

```diff
@@ -41,39 +42,56 @@ def handler(event, context):
     # Load settings
     SETTINGS = BossSettings.load()
 
-    # Load the project info from the chunk key you are processing
-    proj_info = BossIngestProj.fromSupercuboidKey(event["chunk_key"])
-    proj_info.job_id = event["ingest_job"]
-
-    # Get message from SQS ingest queue, try for ~2 seconds
-    rx_cnt = 0
-    msg_data = None
-    msg_id = None
-    msg_rx_handle = None
-    while rx_cnt < 6:
-        ingest_queue = IngestQueue(proj_info)
-        msg = [x for x in ingest_queue.receiveMessage()]
-        if msg:
-            msg = msg[0]
-            print("MESSAGE: {}".format(msg))
-            print(len(msg))
-            msg_id = msg[0]
-            msg_rx_handle = msg[1]
-            msg_data = json.loads(msg[2])
-            print("MESSAGE DATA: {}".format(msg_data))
-            break
-        else:
-            rx_cnt += 1
-            print("No message found. Try {} of 6".format(rx_cnt))
-            time.sleep(1)
+    # Used as a guard against trying to delete the SQS message when lambda is
+    # triggered by SQS.
+    sqs_triggered = False
```

Comment:
should the boolean be this?
sqs_triggered = 'Records' in event and len(event['Records']) > 0